docs/changelog.rst

```diff
@@ -20,6 +20,7 @@ Changelog
 * :feature:`-` Swaps made via uniswap v3 auto routers (both v1 and v2) will now be decoded correctly.
 * :feature:`3231` Optimism is now supported. Optimism balances will be shown and optimism transactions will be decoded.
 * :feature:`1756` Uniswap and sushiswap pool join/exit events are now properly decoded and taken into account during PnL report.
+* :bug:`-` PnL report will now respect the profit currency value for trades when the price for assets is calculated instead of queried.
```

Comment:
so it was not only for manual trades? If it's only manual trades state it as it's much easier to understand from the user perspective